framework/source/class/qx/lang/normalize/Object.js

```diff
@@ -58,6 +58,27 @@ qx.Bootstrap.define("qx.lang.normalize.Object", {
       }
 
       return arr;
+    },
+
+
+    /**
+     * Determines whether two values are the same value.
+     *
+     * <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/is">MDN web docs: Object.is()</a>
+     *
+     * @signature function(value1,value2)
+     * @param value1 {Object} the first value to compare
+     * @param value2 {Object} the second value to compare
+     * @return {Boolean} indicating whether or not the two arguments are the same value.
+     */
+    is : function(v1, v2) {
```

Comment:
This function is different than the polyfill suggested at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/is. Given that it is exercising edge cases, we run the risk of doing something not quite right if we don't use what is suggested. Is there something that this is intended to accomplish that the polyfill from there doesn't?

Reply:
Yes! Noticed that now. My code is from the `german` version of MDN which obviously differs from the en-US version. See: https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Object/is#Polyfill
I'll change it to the one suggested on the en-Us version now.

Reply:
The difference between both versions tells me to allways switch to the en-Us version when searching in MDN, as the german pages seem outdated. :-(
That's odd.